README.md

```diff
@@ -1,100 +1,70 @@
+# Nutrivurv API Documentation
 
 ![Code Maintainability](https://api.codeclimate.com/v1/badges/9ac982aab59ee2c30f71/maintainability)
 
 ![Test Coverage](https://api.codeclimate.com/v1/badges/9ac982aab59ee2c30f71/test_coverage)
 
-  
-
-# API Documentation
-
-  
-
 ## Backend deployed at [Heroku](https://labspt7-nutrition-tracker-be.herokuapp.com/)
 
-  
-
 ## Getting started
 
-  
-
 To get the server running locally:
 
-  
+ -Prerequisites:
+  Install [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/) if you are running Windows 10 Home Edition pre-2004 upate.
 
-- Clone this repo.
+  Install Docker For Desktop if you are running [Windows 10 Home post-2004 update](https://docs.docker.com/docker-for-windows/install-windows-home/), [Pro, Enterprise or Education](https://docs.docker.com/docker-for-windows/install/), or [macOS](https://docs.docker.com/docker-for-mac/install/)
```

Comment:
Awesome 🎉  Loved that you got a jump on updating the documentation. This will be a big help for our's and future teams 👍